ui/src/app/views/notebook-redirect/component.ts

```diff
@@ -30,6 +30,25 @@ enum Progress {
   Redirecting
 }
 
+export enum Kernels {
+  R,
+  Python3,
+}
+
+const rNotebookFileContent = '{ "cells": [ { "cell_type": "code", "execution_count": null, \
```

Comment:
How did you generate this? Is it minimal? I think we should try to minimize any information being encoded here so we don't go out of sync with Leo / the Jupyter server.

For example, what happens if you just reduced this to `{kernelspec: {name: "ir"}}` or `{language_info: {name: "R"}}`

Reply:
I will check if we can do this, but my understanding is that it will break if we provide any less than that... 

I actually very much wish we could just pass the kernel in... but I couldn't figure out how to do that.

Reply:
I generated this by downloading an empty notebook and copying the format. The issue is that we can't use notebook as the type and pass in content. I can show you in person tomorrow or Wednesday if you like.